rusk/build.rs

```diff
@@ -137,38 +136,30 @@ mod transfer {
             &self,
         ) -> Result<(Vec<u8>, Vec<u8>), Box<dyn std::error::Error>> {
             let pub_params = &PUB_PARAMS;
-            let rng = &mut rand::thread_rng();
+            let rng = &mut StdRng::seed_from_u64(0xbeef);
```

Comment:
0xbeef

Reply:
Yes, its preferable to use seeded rngs instead of entropic ones since determinism is desirable here.

Reply:
Oh, I understand. I just giggle with the `0xbeef` thing.